pycep/bicep.lark

```diff
@@ -346,14 +348,28 @@ multiply: value_brackets "*" value_brackets
 
 substract.1: value_brackets "-" value_brackets
 
+// operators - accessor
+
+?operator_accessor: index_accessor | function_accessor | nested_resource_accessor | property_accessor
+
+index_accessor: value_brackets "[" value_brackets "]"
+
+function_accessor: value_brackets "." LIST_FUNC_NAME "(" value_brackets? ["," value_brackets]+ ")"
+
+nested_resource_accessor: value_brackets "::" value_brackets
+
+// 'sys.' and 'az.' have to be eliminated in 1st operand (see STRING definition regexp):
+property_accessor: value_brackets "." value_brackets
+
 // Terms
 
 EXISTING: "existing"
 
-STRING: /[a-zA-Z_](::|[\w_.])*/
+STRING: /(?!(sys|az)\.)[a-zA-Z_]([\w_])*/
 QUOTED_STRING: "'" STRING "'"
-INDEXED: /(?=[\w][\w_.]+(.(get|list)[a-zA-Z]*\(([\w., '-]+)?\)|[\[\]])+)(.(get|list)[a-zA-Z]*\(([\w., '-]+)?\)|\[[\w\[\]_().']+\]!?|[\w_\-\/.,'= ])+/
+INDEXED: /(?=[\w][\w_.]+(.(get|list)[a-zA-Z]*\(('[\w-]+'|[\w.]+)?\)|[\[\]])+)(.(get|list)[a-zA-Z]*\(('[\w-]+'|[\w.]+)?\)|\[[\w\[\]_().']+\]|[\w_\-\/.,'= ])+/
```

Comment:
does this terminal still make sense? probably the `get...` is still a valid use case?

Reply:
This regex is still not optimal, I experimented to simplify it, but broke the parsing, so left this change.
INDEXED is currently used only for **load_json_content** and **json_func** and I doubt it is really needed there.
Maybe it can be improved in a separate PR.